vng_api_common/utils.py

```diff
@@ -6,12 +7,20 @@
 from django.db import models
 from django.http import HttpRequest
 from django.urls import Resolver404, get_resolver
+from django.utils.module_loading import import_string
+
+from zds_client.client import ClientError
+
+from .models import APICredential
 
 try:
     from djangorestframework_camel_case.util import underscore_to_camel as _underscore_to_camel
 except ImportError:
     from djangorestframework_camel_case.util import underscoreToCamel as _underscore_to_camel
 
+
+logger = logging.getLogger(__name__)
```

Comment:
I'm missing an `import logging` statement, no?

Reply:
it's a first line in file (https://github.com/VNG-Realisatie/vng-api-common/pull/37/files#diff-afcf04f01400eb678b852886a57ec9a2R1)

Reply:
oh darn it, I expanded the diff more but apparently it missed that line :)